mainmatter.md

```diff
@@ -276,24 +254,28 @@ This is shown in the following example:
        "token_type":"example",
        "expires_in":3600,
        "refresh_token":"tGzv3JOkF0XG5Qx2TlKWIA",
-       "authorization_details":{  
-         "payment":{  
-           "instructedAmount":{  
-             "currency":"EUR",
-             "amount":"123.50"
-           },
-           "debtorAccount":{  
-             "iban":"DE40100100103307118608"
-           },
-           "creditorName":"Merchant123",
-             "creditorAccount":{  
+       "authorization_details":[  
```

Comment:
Can you please also update the URI query parameter example?

Reply:
I don't see the example you're referring to, I thought I'd updated all of them.